webui/src/pages/repositories/repository/settings/general.jsx

```diff
@@ -86,6 +86,16 @@ const SettingsContainer = () => {
                         <Form.Control readOnly value={repo.id} type="text"/>
                     </Col>
                 </Row>
+                {repo.storage_id &&
```

Comment:
Why do we check `repo.storage_id` here 
and `repo.storage_id && repo.storage_id.length` in index.jsx

Reply:
Good catch.
Updated.